Readme.md

```diff
@@ -17,74 +16,65 @@ Usage:
         show this help
 ```
 
-## How to build
-
-```
-npm install
-npm start
-```
-
-The binary is then in `./lib/bs/native/main.native`.
-
 ## Things still to be implemented
 
-- [ ] Interactive code samples!
-  - [ ] First pass - include the compiled javascript for code samples so you can see the result
-  - [ ] Second pass - make the code samples editable, with an in-page compiler so you can experiment
-- [ ] Parse the bsconfig.json and package.json of a project
-  - [x] to determine the github url
-    - [ ] support all types of github specifications https://docs.npmjs.com/files/package.json
-  - [ ] name, source directories, etc.
-- [ ] Handle ocamldoc (see included octavius source) so we can build ocamly projects too
+* [ ] Interactive code samples!
```

Comment:
why this change?

Reply:
ah that's what happened when it ran through Prettier. It has no effect, but I'll revert it if you want 👍🏾